CONTRIBUTING.md

```diff
@@ -42,10 +42,11 @@ Steps to submit your pull request:
 
 1. Fork `egjs-flicking` on your repository
 2. Create new branch from your egjs master branch (and be sure always to be up-to-date)
-3. Do your work
-4. Create test code for your work (when is possible)
-5. Run `npm run lint` for linting and Code Conventions (update until without any error or warnings)
-6. Run test code by `npm run test OR npm run test:chrome`.
+3. Before starting a local development, `npm run docs:build` in the root directory to generate API documents.
```

Comment:
This is a sub-item of "submitting PRs", what about separating it as a different category?
Like, "Setting up a local development"

Reply:
I also got the feeling that it was different. However, I thought it was part of the process of doing PR, so I put it in submittings PRs. Nevertheless, would it be better to create another category and put it in?

Reply:
I asked because working with our demo is not essential when creating PRs.
There're other ways to test Flicking in local development, you can use `npm run start` script with `/test/manual` pages.
So I think it can be good when we add info about manual testing methods there.

You can leave the PR as-is, we'll write the rest.

Reply:
Oh I see, thank you

Reply:
I've approved the PR, I'm just advising so don't take my words too hard :)